src/test/java/io/aiven/kafka/connect/gcs/GcsSinkTaskTest.java

```diff
@@ -619,6 +656,41 @@ final void supportStructValuesForClassicJson() {
             readRawLinesFromBlob("topic1-0-30", compression));
     }
 
+    @Test
+    final void supportUnwrappedJsonEnvelopeForStructAndClassicJson() {
+        final String compression = "none";
+        properties.put(GcsSinkConfig.FILE_COMPRESSION_TYPE_CONFIG, compression);
+        properties.put(GcsSinkConfig.FORMAT_OUTPUT_FIELDS_CONFIG, "value");
+        properties.put(GcsSinkConfig.FORMAT_OUTPUT_ENVELOPE_CONFIG, "false");
+        properties.put(GcsSinkConfig.FORMAT_OUTPUT_TYPE_CONFIG, "json");
+
+        final GcsSinkTask task = new GcsSinkTask(properties, storage);
```

Comment:
just let you know. we use JDK 11 so instead of boring  `final GcsSinkTask task` you can use `final var`